target/bin/sedfile

```diff
@@ -11,19 +11,20 @@
 
 set -ueo pipefail
 
-HASHTOOL="sha1sum"
+function __usage { echo "Usage: ${0} -i <replace/delete operation> <file>" ; }
+
+HASHTOOL='sha1sum'
 SKIP_ERROR=0
 
-if [[ $# -lt 3 ]]
+if [[ ${#} -lt 3 ]]
 then
-  echo "Error:  At least, three parameters must be given."
-  echo "Syntax: sedfile -i <replace/delete operation> <file>"
-  echo
+  __usage
+  echo 'At least three parameters must be given' >&2
```

Comment:
Not sure, why you removed "Error:"?

```suggestion
  echo 'Error: At least three parameters must be given' >&2
```

Reply:
PS: The whole old if statement was just fine..

Reply:
Done in [`ed512d0`](https://github.com/docker-mailserver/docker-mailserver/pull/2500/commits/ed512d04830330fda13343f49f8d7621249ac424)